### ISSUE TITLE
One Time Job Schedules don't stay in admin console UI

### ISSUE THREAD

**Issue by TurnItOff-OnAgain** [ironmansoftware/powershell-universal#3124]
### Version

4.2.9

### Severity

Low

### Steps to Reproduce

Created a one time schedule for a script. It shows up correctly when first created, but within a few minutes the schedule disappears from the admin UI. It still shows in Hangfire as a scheduled job.

I see there was a one time schedule fix done in 4.2.10, but we do not use git sync or SQL persistence. Also the jobs aren't really deleted since they still show as scheduled in Hangfire.

### Expected behavior

```console
One Time Schedule stays in the admin UI until it runs once
```

### Actual behavior

```console
One Time Schedule disappears a few minutes after being created, but still shows in Hangfire as scheduled
```

### Additional Environment data

4.2.9

### Visuals

![image](https://github.com/ironmansoftware/issues/assets/26926284/b39cd4f1-6c88-4dec-a74c-afcd57aae557)

![image](https://github.com/ironmansoftware/issues/assets/26926284/46f20059-10c8-45e9-91d8-44e737bdfa4e)

![image](https://github.com/ironmansoftware/issues/assets/26926284/5e7e9734-61d2-4f8c-9697-9e450b10844f)

**Comment by TurnItOff-OnAgain** [comment]
Maybe too soon to say, but I upgraded to 4.2.12, recreated the job, and it is still showing in both the admin console and hangfire. Been about 5 minutes now since job creation and it's usually gone by now.

**Comment by TurnItOff-OnAgain** [comment]
Yep, it's fixed with 4.2.12.
